### PR TITLE
Solidity enum style fix

### PR DESCRIPTION
**Pull request by LefterisJP**

### CODE REVIEW

File: libsolidity/AST.cpp [hunk 1]
```diff
@@ -449,9 +449,9 @@ void Expression::requireLValue()
 
 void UnaryOperation::checkTypeRequirements()
 {
-	// INC, DEC, ADD, SUB, NOT, BIT_NOT, DELETE
+	// INC, DEC, ADD, SUB, NOT, BitNot, DELETE
```
Comment (chriseth) [inline libsolidity/AST.cpp]: Forgot some names in the comment here.

File: libsolidity/ExpressionCompiler.cpp [hunk 2]
```diff
@@ -596,18 +596,18 @@ void ExpressionCompiler::appendCompareOperatorCode(Token::Value _operator, Type
 
 		switch (_operator)
 		{
-		case Token::GTE:
+		case Token::GreaterThanOrEquals:
```
Comment (chriseth) [inline libsolidity/ExpressionCompiler.cpp]: I would drop the `s` in the `Equals`. In `GreaterThanOrEquals` it sounds especially weird. Furthermore, `Token::Equals` denotes the symbol and not the relation.

File: libsolidity/Token.cpp [hunk 3]
```diff
@@ -79,8 +79,8 @@ int8_t const Token::m_precedence[NUM_TOKENS] =
 char const Token::m_tokenType[] =
 {
 	TOKEN_LIST(KT, KK)
-};
```
Comment (chriseth) [inline libsolidity/Token.cpp]: Wrong blank lines.

File: libsolidity/Token.h [hunk 4]
```diff
@@ -182,162 +177,162 @@ namespace solidity
...
-	K(INT24, "int24", 0)                                               \
-	K(INT32, "int32", 0)                                               \
-	K(INT40, "int40", 0)                                               \
+	K(Int, "int", 0)                                                   \
+	K(Int8, "int8", 0)                                                 \
+	K(Int16, "int16", 0)                                               \
+	K(Int24, "int24", 0)                                               \
+	K(Int32, "int32", 0)                                               \
+	K(Int40, "int40", 0)                                               \
 	K(INT48, "int48", 0)                                               \
```
Comment (chriseth) [inline libsolidity/Token.h]: You missed one :-)

File: libsolidity/Token.h [hunk 5]
```diff
@@ -182,162 +177,162 @@ namespace solidity
...
+	K(Int192, "int192", 0)                                             \
+	K(Int200, "int200", 0)                                             \
+	K(Int208, "int208", 0)                                             \
+	K(Int216, "int216", 0)                                             \
+	K(Int224, "int224", 0)                                             \
+	K(Int232, "int232", 0)                                             \
+	K(Int240, "int240", 0)                                             \
+	K(Int248, "int248", 0)                                             \
+	K(Int256, "int256", 0)                                             \
+	K(Uint, "uint", 0)                                                 \
```
Comment (chriseth) [inline libsolidity/Token.h]: `UInt`?

File: libsolidity/Token.h (hunk 4 above)
Comment (LefterisJP) [inline libsolidity/Token.h]: wow .. I wonder how did that happen. The Regex missed?

**Comment by LefterisJP** [comment]
Fixed the `Type` enum value inside `Type` class windows error. The reason it currently fails is what Marek talked about with boost time library dependency and is totally unrelated.

**Pull request merged**